tests/model_serving/model_server/keda/test_isvc_keda_scaling.py

```
@@ -0,0 +1,121 @@
+            RunTimeConfigs.ONNX_OPSET13_RUNTIME_CONFIG,
+            {"name": ModelFormat.ONNX, "model-version": ModelVersion.OPSET13, "model-dir": "test-dir"},
+        )
+    ],
+    indirect=True,
+)
+class TestOVMSKedaScaling:
+    def test_ovms_keda_scaling(
+        self,
+        unprivileged_client: DynamicClient,
+        ovms_keda_inference_service: Generator[InferenceService, Any, Any],
+        ovms_pod_resource: Pod,
```

Pylint (3.3.7)

[warning] Unused argument 'ovms_pod_resource'

(W0613)